src/components/TaskBoard/TaskBoard.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { addTask } from "src/store/task/actions/taskActions";
+import TaskNavBar from "src/components/TaskNavBar/TaskNavBar";
+import AddTaskCard from "src/components/AddTaskCard/AddTaskCard";
+import TaskList from "src/components/TaskList/TaskList";
+import "./TaskBoard.scss";
+
+function TaskBoard() {
+  const [isCreateButtonClicked, setIsCreateButtonClicked] = useState(false);
+
+  const tasks = useSelector((state) => state.task.tasks);
```

Comment:
To increase readability can we change redux store so that we can use the line like this.
```suggestion
  const tasks = useSelector((state) => state.tasks);
```

Reply:
@rsa-wtag vai, as we are setting up a way of combining multiple reducers, each reducer must have a property name in the object passed to the `combineReducers` function. So, `task` is the `taskReducer` and `tasks` is the array in initialState that contains all the tasks. So, I thinks it's not possible to remove that term since we are using `combineReducers`.

Reply:
You can do it using an array instead of objects in the taskreducers.js
`const initialTasksState = {
  tasks: [],
};`
`const initialTasksState =  [];`